<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DeviceReport</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icon/about/res/logo.png</normaloff>:/icon/about/res/logo.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="hlayoutLists">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeWidget" name="listwDevices">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>Devices</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="listwDeviceInfo">
      <column>
       <property name="text">
        <string>Field</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actRefresh"/>
    <addaction name="actSave"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menuCategory">
    <property name="title">
     <string>Categories</string>
    </property>
    <addaction name="actAudio_Input_Output"/>
    <addaction name="actDisplays_adapters"/>
    <addaction name="actMonitors"/>
    <addaction name="actImage_processing_devices"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actHelp"/>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCategory"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actRefresh"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/main/res/Quit.png</normaloff>:/icon/main/res/Quit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actAudio_Input_Output">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Audio Input&amp;Output</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actDisplays_adapters">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display adapters</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actMonitors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Monitors</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actImage_processing_devices">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Image processing devices</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actRefresh">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/main/res/Refresh.png</normaloff>:/icon/main/res/Refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actHelp">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/main/res/Help.png</normaloff>:/icon/main/res/Help.png</iconset>
   </property>
   <property name="text">
    <string>View Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/main/res/About.png</normaloff>:/icon/main/res/About.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/main/res/Save.png</normaloff>:/icon/main/res/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
